Dockerfile

```diff
@@ -1,7 +1,13 @@
 FROM ubuntu:16.04
 
-RUN apt-get update && apt-get install -y curl  libunwind8 libicu55
-RUN curl -LO https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.9/powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
-RUN dpkg -i powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
+ENV POWERSHELL_VERSION="6.0.0-alpha.9"
 
-Entrypoint [ "powershell" ]
+RUN apt-get update && \
+    apt-get install -y curl libunwind8 libicu55 && \
+    curl -LO https://github.com/PowerShell/PowerShell/releases/download/v"$POWERSHELL_VERSION"/powershell_"$POWERSHELL_VERSION"-1ubuntu1.16.04.1_amd64.deb && \
+    dpkg -i powershell_"$POWERSHELL_VERSION"-1ubuntu1.16.04.1_amd64.deb && \
```

Comment:
I don't think sudo belongs in the script. You can also use sudo to invoke the script.

I would also change the download script to delete the $package.

Also since the cleanup would be in the Docker file are we going to have multiple Dockerfiles for different distros?